pkg/streamingpromql/testdata/upstream/functions.test

```diff
@@ -1352,25 +1346,20 @@ load 10s53ms
 	metric_histogram{type="only_histogram"} {{schema:1 sum:2 count:3}}x4
 	metric_histogram{type="mix"} _ 1 1 1 {{schema:1 sum:2 count:3}} {{schema:1 sum:2 count:3}} 1
 
-# Unsupported by streaming engine.
-# eval instant at 90s ts_of_first_over_time(metric[90s])
-# 	{} 20.106
+eval instant at 90s ts_of_first_over_time(metric[90s])
+	{} 20.106
 
-# Unsupported by streaming engine.
-# eval instant at 95s ts_of_first_over_time(metric[90s])
-# 	{} 20.106
+eval instant at 95s ts_of_first_over_time(metric[90s])
+	{} 20.106
 
-# Unsupported by streaming engine.
-# eval instant at 15s ts_of_first_over_time(metric[90s])
-# 	#empty
+eval instant at 15s ts_of_first_over_time(metric[90s])
+	#empty
 
-# Unsupported by streaming engine.
-# eval instant at 95s ts_of_first_over_time(metric_histogram{type="only_histogram"}[90s])
-# 	{type="only_histogram"} 10.053
+eval instant at 95s ts_of_first_over_time(metric_histogram{type="only_histogram"}[90s])
+	{type="only_histogram"} 10.053
 
-# Unsupported by streaming engine.
-# eval instant at 95s ts_of_first_over_time(metric_histogram{type="mix"}[90s])
-# 	{type="mix"} 10.053
+eval instant at 95s ts_of_first_over_time(metric_histogram{type="mix"}[90s])
+	{type="mix"} 10.053
```

Comment:
Similar to above - could you please add a test for the case where there are both histograms and floats in the range and the histogram is the first point?